frontend/src/components/EdgeCanvas.tsx

```diff
@@ -3,42 +3,8 @@ import LineChart from 'react-svg-line-chart';
 import {classes, stylesheet} from 'typestyle';
 import {LineageCardType} from './LineageTypes';
 import './LineChart.d.ts';
-
-const px = (number: number): string => `${number}px`;
-
-const edgeWidth = 200;
-const cardBodyHeight = 67;
-const cardBodyWidth = 260;
-
-const cardContainerSpacerHeight = 24;
-const cardTitleHeight = 41;
-const cardTitleBorders = 2;
-const cardOffset = cardContainerSpacerHeight + cardTitleHeight + cardTitleBorders;
-
-const css = stylesheet({
-  edgeCanvas: {
-    border: 0,
-    display: 'block',
-    height: px(440),
-    marginLeft: px(cardBodyWidth),
-    marginTop: px(74),
-    overflow: 'visible',
-    position: 'absolute',
-    width: px(edgeWidth),
-    zIndex: -1,
-    '$nest': {
-      svg: {
-        display: 'block',
-        overflow: 'visible',
-        position: 'absolute',
-      }
-    }
-  },
-  edgeCanvasReverse: {
-    marginLeft: 0,
-    transform: 'translateX(-100%)',
-  },
-});
+import {px} from '../Css';
```

Comment:
This is going to make it challenging to isolate the package to its own repo later. We could discuss this more later